src/controllers/editProfileControllers.js

```diff
@@ -0,0 +1,30 @@
+ require('dotenv').config();
+ const User = require('./../models/user')
+ const {ERROR, SUCCESS} = require('.././assests/constants')
+
+
+
+const editProfile = async (req, res) => {
+    try {
+        const id = req.params.id;
+        const userData = await User.findById({userId: id});
+        if (userData) {
+            res.status(SUCCESS).json({
+                message:'userData found successfully',
+                data:userData
+            })
+        }else {
+            res.status(ERROR).json({
+                message: 'Wrong data found'
+            })
+        }
+        
+    }catch (err) {
+        res.status(ERROR).json({
+            message: err.message
+        })
```

Comment:
inside catch, all you need to do is next(err)....pass error into next function so that the universal error handler will take care of it